### PR TITLE
Remove hard-coded -xSSE4.2 for numpy/scipy with intel compiler

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: easybuild/easyconfigs/p/Python/Python-3.5.1-intel-2016a.eb [hunk 1]
```diff
@@ -41,6 +41,7 @@ exts_list = [
         'source_urls': ['https://pypi.python.org/packages/source/n/numpy/'],
         'patches': [
             'numpy-1.8.0-mkl.patch', # % numpyversion,
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-3.5.1-intel-2016a.eb]: please remove the `# % numpyversion` while you're at it

**Comment by boegel** [comment]
@hajgato: why only for Python 3.5.1?

**Comment by hajgato** [comment]
@boegel: Other configs with numpy 1.10 included. (numpy 1.9 is safe)

**Comment by boegel** [comment]
@hajgato: same for numpy 1.10.x easyconfigs?

```
numpy-1.10.1-intel-2015b-Python-2.7.10.eb
numpy-1.10.4-intel-2015b-Python-2.7.10.eb
numpy-1.10.4-intel-2016a-Python-2.7.11.eb
```

**Comment by hajgato** [comment]
@boegel: hmmm.. added

**Comment by boegel** [comment]
lgtm, submitted a test report to make sure...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6cf6707261ce6b9e1e1f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c39edb647bb1c9f21f55 for a full test report.

**Comment by boegel** [comment]
lgtm, thanks @hajgato!

**Pull request merged**